src/js/register.jsx

```diff
@@ -118,6 +118,7 @@ function Register() {
             </label>
             <LanguageButtons selectedLanguage={defaultLang} selectLanguage={setDefaultLang} />
           </div>
+          <hr style={{ border: "1px dashed", borderColor: "var(--gray" }} />
```

Comment:
We can also use emotion inlined class css with `<Component css={css`border-color: red;`}`>  or with  `<Component css={{borderColor: red}}/>` `after importing `import { css } from '@emotion/react'` for consistency. https://emotion.sh/docs/@emotion/react